### ISSUE TITLE
Conflict: Multiple assets emit different content to the same filename index.html

### ISSUE THREAD

**Issue by YannickSilva** [ionic-team/ionic-framework#25174]
Hello i started a new project with Ionic Vue and whenever i execute ionic serve it's returns me this error saying: Conflict: Multiple assets emit different content to the same filename index.html

But it's a fresh project i didn't touch in anything yet, i have been trying to fix this bug but without success
![ScreenShot_20220422144301](https://user-images.githubusercontent.com/25250749/164726473-8ce61d4c-43f2-4612-b4d4-93a5de4b7054.png)
.

**Comment by sean-perkins** [comment]
@YannickSilva did you create a new project with the Ionic CLI? Using `ionic start` and generating a Vue project from our available starters? If so, can you provide which starter template you selected?

**Comment by YannickSilva** [comment]
Yes, I created a new project using Ionic cli and ionic start to generate a Vue Project, and I selected the blank starter template.

**Comment by sean-perkins** [comment]
I've locally confirmed that the Ionic Vue blank starter serves correctly on macOS (Monterey) and also Amanda confirmed the same on a Windows machine. 

Could you provide additional information about your development environment that may help us triage the issue?
- node & npm version 
- Do you have the Vue CLI installed globally? If so, which version?

Other developers have reported this issue on the forums recently: https://forum.ionicframework.com/t/webpack-compilation-error-mobile-app-with-tabs/219673, so there is likely an issue somewhere here.

Thanks!

**Comment by YannickSilva** [comment]
Node - v18.0.0
Npm- 8.1.2
Vue-Cli - 5.0.4

**Comment by liamdebeasi** [comment]
@YannickSilva Does this happen if you create a blank Vue app without using Ionic?

Try following the steps on https://cli.vuejs.org/#getting-started to create a Vue project.

**Comment by YannickSilva** [comment]
I will upload a project that gives me this error and share it, it's good for you ?

**Comment by liamdebeasi** [comment]
That would be great, thank you.
